src/en/charms-deploying.md

```diff
@@ -367,55 +367,17 @@ official Charm Store. Such cases include:
 - The charms may exist online but they have been customized locally.
 
 !!! Note: 
-    Although this method will ensure that the charms themselves are
-    available on systems without outside internet access, there is no guarantee
-    that a charm will work in a disconnected state. Some charms will attempt to pull
-    code from sources on the internet such as GitHub.
+    Although this method will ensure that the charms themselves are available
```

Comment:
Can you turn off your 'auto-wrapping' setting or whatever - it just generates noise in the diffs

Reply:
I don't understand.